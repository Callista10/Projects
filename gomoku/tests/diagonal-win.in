place A1
place J2
place B2
place H3
place C3
place H5
place D4
place E12
place E5
