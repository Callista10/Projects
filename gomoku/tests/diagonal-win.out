Black wins!
A1J2B2H3C3H5D4E12E5
Thank you for playing!
